spec2vec/vector_operations.py

```diff
@@ -41,12 +45,14 @@ def _check_model_coverage():
             weights_missing_raised = numpy.power(weights_missing, intensity_weighting_power)
             missing_percentage = 100 * weights_missing_raised.sum() / (weights_raised.sum()
                                                                        + weights_missing_raised.sum())
-            print(f"Found {len(idx_not_in_model)} word(s) missing in the model.",
-                  f"Weighted missing percentage not covered by the given model is {missing_percentage:.2f}%.")
+            msg = (f"Found {len(idx_not_in_model)} word(s) missing in the model.",
+                   f"Weighted missing percentage not covered by the given model is {missing_percentage:.2f}%.")
+            logger.info(msg)
```

Comment:
I am not very familiar with logging, but if adding the message with the info function can be easily filtered out, I think it makes sense to always print this. If it is difficult to filter this out, I would only print it if it is above the set allowed_missing_percentage, because it will otherwise fill up the logging quickly and make you miss other logging statements.
Now another solution pops to my mind that would be even better (but probably not worth the effort). It might be more useful to at the end just add a summary to the logging. Where you print the distribution of missing percentages in some way. The main problem is not that this information is returned, but that it is returned as separately printed lines, since if you do this for a few 1000 spectra, you will not be able to quickly check if there are any issues.  But a summary like: 
total spectra: 1000
missing 0%: 800 spectra
missing 0-10% 100 spectra
missing 5-10% 50 spectra
missing 10-100% 50 spectra
Would actually be very useful, since this quickly shows you if the model you use is useful for your dataset.

Reply:
In logging you can have different logging levels: DEBUG, INFO, WARNING, ERROR, CRITICAL
The default setting of the spec2vec logger is "WARNING", so it will only log everything on the level of warning or above. If users want to record all information available during the process they can change the setting by
```
from spec2vec import set_spec2vec_logger_level
set_spec2vec_logger_level("INFO")
```
So, it will only be recorded if users actively switch this on.

Reply:
The more global summary of results would be useful indeed. We should make a new issue for that!